tests/core/test_stats.py

```diff
@@ -167,6 +166,7 @@ def test_does_send_stats_using_dogstatsd_with_tags(self):
         )
 
     def test_does_send_stats_using_dogstatsd_when_statsd_and_dogstatsd_both_on(self):
+        # ToDo: Figure out why it identical to test_does_send_stats_using_dogstatsd_when_dogstatsd_on
         self.dogstatsd.incr("empty_key")
         self.dogstatsd_client.increment.assert_called_once_with(
             metric='empty_key', sample_rate=1, tags=[], value=1
```

Comment:
Just add todo to figure out later why implementation same as `test_does_send_stats_using_dogstatsd_when_dogstatsd_on`